bedrock/base/templates/base-new.html

```diff
@@ -7,13 +7,13 @@
   <head>
     <meta charset="utf-8">{# Note: Must be within first 512 bytes of page #}
 
+    {% block experiments %}{% endblock %}
```

Comment:
Perhaps this should come after the main javascript `site` bundle? Doing it as early as possible is good, but if we want to give people the opportunity to opt-out of experiments (using DNT for example), then moving it after `site` bundle would enable that.

As a side thought, should it be up to each experiment to honor DNT, or should we bake this into the traffic cop lib itself? Open to suggestions as to what you think is best.